Tests/L0/MSBuild/ParsesWithoutError.ps1

```diff
@@ -1,8 +0,0 @@
-[cmdletbinding()]
```

Comment:
This test is totally unnecessary now since many other tests are in place now that load up the files (guaranteeing they are parsable).